### ISSUE TITLE
Problem using async&await

### ISSUE THREAD

**Issue by hosein2398** [rollup/rollup-plugin-babel#209]
Hi there,
I'm using async in my code and really wonder how to put it to work this plugin, cause I tried MANY ways and I get this error when my code is compiled : 
```
uncaught ReferenceError: regeneratorRuntime is not defined
```
I'm using env preset in my config, Here is my config:
```
{
  "presets": [
    ["env", {
      "modules": false,
      "targets": {
        "browsers": ["last 8 versions"]
      }
    }]
  ],
  "plugins": ["external-helpers"]
}
```
should this be enough for using async/await with babel ? Cause as it's said in Babel doc this preset will led to `@babel/preset-es2017` which i guess async is part of it.

But with this confiig I get the above error, and more interestingly my async functions are compiled to generators in my build file those change to something like:
```
var asyncToGenerator = function (fn) {
    return function () {
      var gen = fn.apply(this, arguments);
      return new Promise(function (resolve, reject) {
        function step(key, arg) {
          try {
            var info = ge....
```
 Since I couldn't get the wanted result this way I tried `babel-plugin-transform-async-to-generator` and `babel-plugin-transform-regenerator` and `babel-polyfill` but just nothing!

I'm using : 
```
"babel-core": "^6.26.0",
"babel-preset-env": "^1.6.1",
"rollup": "^0.57.1",
"rollup-plugin-babel": "^3.0.3",
"rollup-plugin-node-resolve": "^3.3.0"
```

**Comment by Andarist** [comment]
U can add:
```js
import 'regenerator-runtime/runtime'
```
to your main entry point, this will make `regeneratorRuntime` a global though. Somewhat better solution is to use `babel-plugin-transform-runtime` appropriately configured.

**Comment by aendrew** [comment]
@Andarist Okay, but that's not a very explicit answer even though you're pointing other issues at it. Would you mind please going into slightly more detail about how one might appropriately configure `babel-plugin-transform-runtime` in the context of your plugin? Thanks.

**Comment by aendrew** [comment]
If "use `babel-plugin-transform-runtime` appropriately configured" is a bit oblique to anyone arriving at this issue:

1. `$ npm i babel-plugin-transform-runtime`
2. Here's my Rollup `plugins` stanza:
```
  plugins: [
    babel({
      babelrc: false,
      runtimeHelpers: true,
      exclude: 'node_modules/**',
      presets: [['env', { modules: false }], 'react'],
      plugins: ['external-helpers', 'transform-runtime', 'transform-object-rest-spread'],
    }),
```

N.b., this is a React context, you might want to remove the `react` preset and the `transform-object-rest-spread` plugin, depending on your use case.

**Comment by mesqueeb** [comment]
@Andarist I am having the same issue where I can not run any files compiled with async functions.

Error is the same `uncaught ReferenceError: regeneratorRuntime is not defined`.

#### My rollup plugins:
```js
plugins = [
  babel({
    runtimeHelpers: true,
    exclude: 'node_modules/**' // only transpile our source code
  }),
  commonjs()
]
```
#### My `.babelrc`:
```js
{
  "presets": [
    ["@babel/preset-env", {
      "modules": false
    }]
  ],
  "plugins": [
    "@babel/plugin-external-helpers",
    "@babel/plugin-proposal-object-rest-spread"
  ]
}
```
I tried tweaking my settings more similar to @aendrew but it didn't work. I think he uses a completely different babel version.

Do you have any idea how to make rollup compile my files without using this undefined `regeneratorRuntime`? Thanks!

**Comment by jcubic** [comment]
I have this and it don't work, the regnerator is not included:
```javascript
export default {
    input: "src/main.js",
    output: {
        name: "lib",
        file: "dist/bundle.js",
        format: "iife"
    },
    plugins: [
        babel({
            "babelrc": false,
            "runtimeHelpers": true,
            "plugins": [
                "@babel/plugin-external-helpers",
                "transform-async-to-generator",
                "syntax-async-functions",
                "@babel/plugin-transform-runtime"
            ],
            "presets": [
                "@babel/preset-env"
            ]
        }),
        commonjs()
        
        //nodeResolve({
        //  jsnext: true,
        //  main: false
        //})
    ]
};
```

**Comment by jcubic** [comment]
I've also tried this:
```javascript
                ["@babel/plugin-transform-runtime", {
                    "helpers": true,
                    "regenerator": true
                }]
```

and I've installed `@babel/runtime`

**Comment by Andarist** [comment]
Could u share your repository or a repository with the issue reproduced? Without it it's hard to pinpoint the exact problem.

**Comment by jcubic** [comment]
I've make it kind of work the code and repo is in #252

**Comment by shravya95** [comment]
If helpful, following fixed the problem for me, added ` @babel/plugin-transform-runtime` to plugins
`plugins: [
          '@babel/plugin-transform-runtime'
 ],`

and my package.json  looks like 
`
"dependencies": {
    "@babel/runtime": "^7.1.2",
}
"devDependencies" : {
"@babel/plugin-transform-runtime": "^7.1.0",
}
`

**Comment by tapan2721** [comment]
> If helpful, following fixed the problem for me, added ` @babel/plugin-transform-runtime` to plugins
> `plugins: [ '@babel/plugin-transform-runtime' ],`
> 
> and my package.json looks like
> `"dependencies": { "@babel/runtime": "^7.1.2", } "devDependencies" : { "@babel/plugin-transform-runtime": "^7.1.0", }`

@shravya95 I have been struggling with this for a while. Thanks a lot.

**Comment by jcubic** [comment]
Finally resolved the issue in #306 last time I simply remove async await from the code.
